server/routes/admin.js

```diff
@@ -17,6 +17,15 @@ router.get('/categories', function (req, res, next) {
     categoryClient.listCategory(callback);
 });
 
+router.get('/articles', function (req, res, next) {
+  const callback = (error, article) => {
+     res.render("admin-list-articles",{
+      article
```

Comment:
the variable name should be articles, not article (because it's an array)

Reply:
Also, I would recommend the long notation while we're still learning JavaScript:

```javascript
{
   articles: articles
}